apps/acqdat_core/lib/acqdat_core/streams/streams.ex

```diff
@@ -53,15 +53,16 @@ defmodule AcqdatCore.Streams do
     {:ok, project} = Project.get_by_id(project_id)
     project = Repo.preload(project, :pipelines)
 
-    orignator = get_orignator(data, data["mode"])
-
-    payload = parse_payload(data, data["mode"])
-
-    event = %Event{originator: orignator, timestamps: nil, payload: payload}
-    state = {event, %{}}
-
     case Cache.get_rooted_graph(project.pipelines.id) do
       {:ok, {root_id, graph}} ->
+        orignator =
+          get_orignator(data, data["mode"])
+          |> Map.put(:pipeline_id, project.pipelines.id)
```

Comment:
please remove single piping
```suggestion
          data
          |> get_orignator(data["mode"])
          |> Map.put(:pipeline_id, project.pipelines.id)
```

Reply:
@niteshcode4u Done